src/app/common/utils.ts

```diff
@@ -20,6 +24,25 @@ interface MakeBitcoinTxExplorerLinkArgs {
   bitcoin: BitcoinChainConfig;
 }
 
+export type AssetFilter = 'all' | 'enabled' | 'disabled';
```

Comment:
How about adding this new code in a separate file? In `src/app/common/filter-tokens.ts` or something like that? 

This `utils.ts` file is already really big and we should be gradually splitting it up and moving code out of it. That will also help us to gradually move functionality to our [mono-repo utils package](https://github.com/leather-io/mono/tree/dev/packages/utils).

You could also then maybe add a unit test to make sure that `isTokenEnabled` and `filterTokens` are working as expected

Reply:
fixed all the comments. just need to add the tests after work.